docs/security.rst

```diff
@@ -2,13 +2,29 @@
 
 Security Overview
 -----------------
-The Schema Registry currently supports two security features: communication with a secure Kafka cluster over SSL; and API calls over HTTPS. At this time, ZooKeeper security and Kafka SASL authentication are not yet supported.
+The Schema Registry currently supports all Kafka security features, including: communication and authentication with a
+secure Kafka cluster over SSL; authentication with ZooKeeper over SASL; authentication with Kafka over SASL; and
+end-user REST API calls over HTTPS.
 
 For more details, check the :ref:`configuration options<schemaregistry_config>`.
 
 Kafka Store
 ~~~~~~~~~~~
-The Schema Registry uses Kafka to persist schemas. The following Kafka security configurations are currently supported:
+The Schema Registry uses Kafka to persist schemas, and all Kafka security features are supported by the Schema Registry.
 
-* SSL encryption
-* SSL authentication
+ZooKeeper
+~~~~~~~~~
+ZooKeeper SASL authentication is supported.
+
+Setting up ZooKeeper SASL authentication for the Schema Registry is similar to Kafka's setup. Namely,
+create a keytab for the Schema Registry, create a JAAS configuration file, and set the appropriate JAAS Java properties.
+
+In addition to the keytab and JAAS setup, be aware of the `zookeeper.set.acl` setting. This setting, when set to `true`
+, enables ZooKeeper ACLs, which limits access to znodes.
+
+Important: if `zookeeper.set.acl` is set to `true`, the Schema Registry's service name must be the same as Kafka's, which
+is most likely `kafka`. The Schema Registry's principal must have the correct service name, too. Otherwise, the Schema
+Registry will fail to create the `_schemas` topic, which will cause a leader not available error.
```

Comment:
This isn't critical, but really helpful to users in tracking down errors. Can we specifically catch the `LeaderNotAvailableException` and include a note that it may be due to this mismatched config between ACL configs between Kafka and schema registry?